4_Zadania_dodatkowe/4.ProgressBar/js/app.js

```diff
@@ -14,17 +14,12 @@ $(document).ready(function () {
         var clickedBarId = $(this).data('bar');
         // console.log(clickedBarWidth);
 
-        // barProgress.each(function(){
-        //     if (barProgress.attr("id") === clickedBarId) {
-        //     console.log(clickedBarId);
-        //     }
-        // });
+        barProgress
+            .filter('#'+clickedBarId)
```

Comment:
so actually this can be simplified a little bit
there is no need in creating `barProgress` variable
you can remove line 7 and write `$('#' + clickedBarId)` (as there should be no two elements with the same `id`)